### PR TITLE
SoundWire: rt714 support corrections

### PR DESCRIPTION
**Pull request by plbossart**
@shumingfan @jack-cy-yu @bardliao @libinyang do the following patches make sense?

### CODE REVIEW

**Comment by jack-cy-yu** [comment]
@plbossart I'm okay with these two patches.

**Review by bardliao — APPROVED** [review]
It makes sense to me

File: sound/soc/intel/boards/sof_sdw.c [hunk 1]
```diff
@@ -295,12 +295,28 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 	},
 	{
 		.part_id = 0x714,
+		.version_id = 3,
+		.direction = {false, true},
+		.dai_name = "rt715-aif2",
```
Comment (lyakh) [inline sound/soc/intel/boards/sof_sdw.c:300]: I see it was like this before, but just double-checking: the "715" DAI name is correct for 714 too?

Comment (bardliao) [reply sound/soc/intel/boards/sof_sdw.c:300]: > I see it was like this before, but just double-checking: the "715" DAI name is correct for 714 too?

That is a good question. The dai name comes from codec driver. So, yes, the dai name is "rt715-aif2" for both rt715 and rt714 codecs

**Comment by plbossart** [comment]
> @plbossart I'm okay with these two patches.

do you mind approving the PR @jack-cy-yu, that will help us keep track of your review when submitting upstream.

**Review by jack-cy-yu — APPROVED** [review]

File: sound/soc/intel/boards/sof_sdw.c (hunk 1 above)
Comment (lyakh) [reply sound/soc/intel/boards/sof_sdw.c:300]: @bardliao thanks for the explanation! So, ideally that driver should be called "rt714-rt715-aif2"? Not hugely important, and nothing to stop this patch of course.

**Review by lyakh — APPROVED** [review]

**Pull request merged**